.travis/smoke-test.sh

```diff
@@ -21,7 +21,7 @@ services=(
     web
 )
 for service in "${services[@]}"; do
-    .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_${service}_1"
+    .scripts/docker/wait-healthy.sh "${COMPOSE_PROJECT_NAME}_${service}_1" 60
```

Comment:
Timeouts are a bit odd here though, as they're from when the script start for the container, not the beginning (so 60 + _x_ seconds).